bin/_test-helpers.sh

```diff
@@ -263,6 +263,11 @@ start_kind_test() {
 }
 
 start_k3d_test() {
+  if [ -n "$RUN_ARM_TEST" ]; then
+    echo "Skipped because ARM tests run on a dedicated cluster."
+    exit 0
```

Comment:
Doesn't `exit 0` exit the script altogether and skip the tests that come after the multicluster one?